src/huggingface_hub/repocard_types.py

```diff
@@ -19,16 +17,34 @@ class ModelIndex:
 class SingleMetric:
     type: str
     """
-    Example: wer
+    Example: wer. Use metric id from https://hf.co/metrics
     """
 
     value: Any
     """
     Example: 20.0 or "20.0 ± 1.2"
     """
 
+    name: Optional[str] = None
+    """
+    Example: Test WER
+    """
+
+    config: Optional[str] = None
+    """
+    The name of the metric configuration used in `load_metric()`. Example: bleurt-large-512 in `load_metric("bleurt", "bleurt-large-512")`. \
+    See the `datasets` docs for more info: https://huggingface.co/docs/datasets/v2.1.0/en/loading#load-configurations
+    """
+
     args: Any = field(default=None)
```

Comment:
Maybe we can take the occasion to document this field as well? E.g. something like

```suggestion
    args: Any = field(default=None)
    """
    The arguments passed during `Metric.compute()`. Example for `bleu`: max_order: 4
    """
```